### DISCUSSION TITLE
Cannot play 2.2 cutscene

### DISCUSSION THREAD

**Discussion by BUnipendix** [BUnipendix/PyCriUsm#14] [Announcements]
my star rail generated function cannot work with latest version key. The latest decrypted video content became corrupted.
![图片](https://github.com/BUnipendix/PyCriUsm/assets/100204822/015427dd-2760-4a24-97bb-fa8460d3244a)

**Comment by comjun04** [comment]
If you're still working on this project, try this key:
6715710760717247 (0x0017dbe7424e8bbf)

**Comment by ghost** [comment]
If u need keygen contact me on discord cyrenehoney
